themes/tdr/login/login-update-password.ftl

```diff
@@ -0,0 +1,34 @@
+<#import "template.ftl" as layout>
+<#assign updatePasswordPageTitle = msg("updatePasswordTitle")>
+
+<@layout.registrationLayout pageTitle=updatePasswordPageTitle; section>
+    <#if section = "header">
+        ${updatePasswordPageTitle}
+    <#elseif section = "form">
+        <form id="kc-passwd-update-form" class="${properties.kcFormClass!}" action="${url.loginAction}" method="post">
+            <input type="text" id="username" name="username" value="${username}" autocomplete="username" readonly="readonly" style="display:none;"/>
+            <input type="password" id="password" name="password" autocomplete="current-password" style="display:none;"/>
```

Comment:
Out of curiosity, what do these hidden input fields do?

Reply:
Unclear. I've removed them and everything still works as expected.